src/lib/php/common-sysconfig.php

```diff
@@ -386,6 +386,12 @@ function Populate_sysconfig()
   $valueArray[$variable] = array("'$variable'", "30", "'$patTokenValidityPrompt'",
     strval(CONFIG_TYPE_INT), "'PAT'", "1", "'$patTokenValidityDesc'", "null", "null");
 
+  $variable = "SourceCodeDownloadRights";
+  $SourceDownloadRightsPrompt = _('Acces rights required to download source code');
+  $SourceDownloadRightsDesc = _('Choose which acces level will be required for user to be able to download source code.');
+  $valueArray[$variable] = array("'$variable'", "'3'", "'$SourceDownloadRightsPrompt'",
+    strval(CONFIG_TYPE_DROP), "'DOWNLOAD'", "1", "'$SourceDownloadRightsDesc'", "null", "'Administrator{10}|Read_Write{3}'");
```

Comment:
Instead of writing `10` and `3`, will it possible to use the values from `Fossology\Lib\Auth\Auth` class?

https://fossology.github.io/d9/dbd/Auth_8php_source.html#l00060

Reply:
Yes, definitely it'd be better - I thought about it but to be honest I eventually forgot to find a class with those values and change it.

Thanks for this tip - I will correct this

Reply:
I guess when you are done with change, you can make the PR ready for review. The approach looks good enough to me.